src/Test/Unit/Config/Validator/Build/ComposerFileTest.php

```diff
@@ -137,10 +155,27 @@ public function testValidateWrongComposerJson(): void
      */
     public function testValidateMagentoLower23(): void
     {
-        $this->magentoVersionMock->expects($this->once())
+        $this->magentoVersionMock->expects($this->exactly(1))
+            ->method('isGreaterOrEqual')
+            ->withConsecutive(['2.3'])
+            ->willReturnOnConsecutiveCalls(false);
+        $this->fileListMock->expects($this->never())
+            ->method('getMagentoComposer');
+        $this->resultFactoryMock->expects($this->once())
+            ->method('success');
+
+        $this->validator->validate();
+    }
+
+    /**
+     * @throws UndefinedPackageException
+     */
```

Comment:
This `@throws` tag should only be on the last method of the class.

Reply:
These docblocks are not blockers necessarily; my only real concern is the invalid test for Magento 2.4.3+